pages/osx/autopkg.md

```diff
@@ -0,0 +1,35 @@
+# autopkg
+
+> Automation framework for macOS software packaging and distribution.
+
+- Run autopkg recipe:
+
+`autopkg run {{recipe_identifier}}`
+
+- Run autopkg with in verbose mode:
```

Comment:
Perhaps change this to `Run autopkg in verbose mode:` (just a minor grammatical change).